### PR TITLE
Improve grammar and diction

### PR DESCRIPTION
**Pull request by gibson042**
* Replace "DST" and "daylight savings time" with "daylight saving time"
* Replace "timezone" with "time zone"
* Properly capitalize "Unicode Consortium"
* Improve phrasing

### CODE REVIEW

File: draft-ietf-sedate-datetime-extended.md [hunk 1]
```diff
@@ -132,23 +132,22 @@ in {{RFC3339}} that has the following properties:
 
 
 This document does not address extensions to the format where the
-semantic result no longer is a fixed timestamp that is referenced to a
-(past or future) UTC time.
+semantic result is no longer an unambiguous instant in time (past or
```
Comment (cabo) [inline draft-ietf-sedate-datetime-extended.md:135]: There is a reason that the existing text says "referenced to UTC".
(I have no idea what "unambiguous" means for timestamps, so I would prefer not to use this word.)

File: draft-ietf-sedate-datetime-extended.md [hunk 2]
```diff
@@ -132,23 +132,22 @@ in {{RFC3339}} that has the following properties:
...
 For instance, it does not address:
 
 * Future time given as a local time in some specified time zone, where
   changes to the definition of that time zone (e.g., a political
-  decision to enact or rescind daylight savings time) changes the
-  actual time in UTC time.
-* Time given as a "floating time", i.e., a local time without
-  information as to which time zone this local time refers to.
+  decision to enact or rescind daylight saving time) affect the
+  instant in time corresponding with the timestamp.
```
Comment (cabo) [inline draft-ietf-sedate-datetime-extended.md:142]: Again, I'd prefer to keep UTC here.

File: draft-ietf-sedate-datetime-extended.md [hunk 3]
```diff
@@ -176,15 +175,15 @@ UTC:
...
   International de l'Heure (BIH).
-  Before 1972 UTC was not generally recognized and civil time was
+  Before 1972, UTC was not generally recognized and civil time was
   determined by individual jurisdictions using different techniques
   for attempting to follow Universal Time based on measuring the
   rotation of the earth.
 
   UTC is often mistakenly referred to as GMT, an earlier timescale
-  UTC was designed to be a useful successor for.
+  that UTC was designed to supersede.
```
Comment (cabo) [inline draft-ietf-sedate-datetime-extended.md:186]: I don't think that UTC (an international time) can aspire to "supersede" GMT (a UK time).  I'd stick with the carefully chosen wording.

File: draft-ietf-sedate-datetime-extended.md [hunk 4]
```diff
@@ -255,15 +253,15 @@ By including a hyphen/minus sign "`-`", the namespace can be separated
...
-For example, if "`u-`" is a namespace for the Unicode consortium, a
-calendar as defined by that consortium could be included as
+For example, if "`u-`" is a namespace for the Unicode Consortium, a
+calendar as defined by that organization could be included as
 `u-ca=<value>`.
 
 An IANA registry for namespaces can be used to allocate namespaces for
 specific applications, as defined in {{iana-cons}}.  Two namespaces are
-allocated by the present document:
+allocated by this document:
```
Comment (cabo) [inline draft-ietf-sedate-datetime-extended.md:262]: I generally don't know what "this document" refers to.
Can we be more explicit?

**Review by cabo — COMMENTED** [review]
Thank you!
I would like to continue to be very explicit about the role of UTC, hence a few comments.
(Also, "this document" is a constant source of confusion, which I try to avoid by being more explicit.)

File: draft-ietf-sedate-datetime-extended.md (hunk 1 above)
Comment (gibson042) [reply draft-ietf-sedate-datetime-extended.md:135]: > There is a reason that the existing text says "referenced to UTC".

Can you articulate that reason? What does it even mean for a timestamp (which is "unambiguous representation of some instant in time", presumably an instance of the `date-time-ext` production) to be "referenced to a UTC time"? The intent of this rewording is to cover hypothetical format extensions where the semantic result of interpreting an instance of `date-time-ext` is "no longer an unambiguous instant in time" but instead describes either no instant in time (such as a local time skipped by DST transition) or multiple instants (such as a local time that is duplicated by DST transition).

File: draft-ietf-sedate-datetime-extended.md (hunk 2 above)
Comment (gibson042) [reply draft-ietf-sedate-datetime-extended.md:142]: What is "actual time in UTC time"? UTC is a time scale organizing instants according to calendar days and clock times within them, so "in UTC time" is clear enough. But "actual time" seems ill-defined to me. How about this?
```suggestion
  instant in UTC time corresponding with the timestamp.
```

File: draft-ietf-sedate-datetime-extended.md (hunk 3 above)
Comment (gibson042) [reply draft-ietf-sedate-datetime-extended.md:186]: As far as I can tell, "supercede" and "successor" are conceptually interchangeable (both referring to the temporal sequence in which distinct entities occupy the same position) and in fact "superseder" is synonymous with "successor" according to https://www.merriam-webster.com/thesaurus/superseder . But perhaps we can find an even better phrasing?
```suggestion
  that predates the introduction of Universal Time and currently
  coincides with a UTC offset of 00:00.
```

**Review by gibson042 — COMMENTED** [review]
Thanks for the review, @cabo! Continued discussion in comment threads.

File: draft-ietf-sedate-datetime-extended.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:135]: RFC 3339 is specifically tied to UTC (see its section 1: https://www.rfc-editor.org/rfc/rfc3339#section-1).  This document [hah] is not intended to change this (future ones might add a timescale parameter).

File: draft-ietf-sedate-datetime-extended.md (hunk 3 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:186]: I have a different dictionary, which does not equate "succeed" and "supersede" as much.
I really think that what the original text says is an accurate description, and trying to wordsmith this in a way that insinuates a relationship beyond the fact that one was inspiring the definition of the other is going to create difficulties.  I also don't want to open the whole can of worms of UT1 etc. here.  So I stand by the original text.

File: draft-ietf-sedate-datetime-extended.md (hunk 4 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:262]: "This document" is fine unless it isn't.  This passage refers to a registry, which (loosely speaking) is a document.  So "this document" is ambiguous (and actually preferably refers to any referent that may be in scope, which is the wrong reference here) and needs to be disambiguated.  Changing the other 13 instances to also be unambiguous may be needed if a candidate reference is in scope, but it is this one that is clearly confusing to this reader.

File: draft-ietf-sedate-datetime-extended.md (hunk 3 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:186]: BTW, https://en.wikipedia.org/wiki/Greenwich_Mean_Time has a nice description of the different ongoing meanings of GMT (mean solar time, civil time in the UK during winter), none of which is actually superseded by UTC.

Comment (gibson042) [reply draft-ietf-sedate-datetime-extended.md:186]: > ongoing meanings of GMT... none of which is actually superseded by UTC

Isn't that in itself an argument against keeping the current "UTC was designed to supersede" language?

Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:186]: I don't know what "current" means here, but the pre-PR language is about UTC being "a useful successor for" GMT.  That is why I would like to keep it.

**Comment by cabo** [comment]
If you don't have time to work further on this very useful PR, I could merge it and fix these three nits.

**Comment by cabo** [comment]
Please don't add material to this one.
We went the opposite direction in the last interim, so I'm about to remove some of this text.

File: draft-ietf-sedate-datetime-extended.md (hunk 4 above)
Comment (gibson042) [reply draft-ietf-sedate-datetime-extended.md]: I've updated this to clearly reference the registry, although I think more dramatic refactoring is in order to initialize the registry with more formal entries that specify all fields of the included template as in e.g. [RFC 7763](https://www.rfc-editor.org/rfc/rfc7763.html#section-6) and [RFC 5545](https://datatracker.ietf.org/doc/html/rfc5545#section-8).

**Comment by gibson042** [comment]
OK, I pushed a commit just before your last comment but will withhold any further changes.

**Comment by cabo** [comment]
Sorry for the unsmooth handling of this PR.
I now cherry-picked from this PR.
Can I ask you to rebase to main and see what remaining parts you want to pursue?

**Comment by cabo** [comment]
My last message was too close to the holidays.
Do you want to rebase this PR?  Or do you think it is essentially covered by my cherry-picking?

**Comment by gibson042** [comment]
Thanks for the ping, @cabo. Almost everything was covered by the cherry-picking, but a few things remain. I have rebased the PR to include only those.

**Pull request merged**